docs/detections/rules-ui-create.asciidoc

```diff
@@ -263,35 +263,100 @@ For example, if a rule has an interval of 5 minutes, no additional look-back tim
 [[create-esql-rule]]
 === Create an ES|QL rule
 
-IMPORTANT: This is a placeholder for future documentation. The following content is incomplete.
+preview::[]
 
-. Go to *Rules* -> *Detection rules (SIEM)* -> *Create new rule*. The *Create new rule* page displays.
-. To create a rule that uses ES|QL, select **ES|QL**, 
-then write a query. There are two types of ES|QL queries:
+Use {ref}/esql.html[ES|QL] to query your source events and aggregate event data. Query results are returned in a table with rows and columns. Each row becomes an alert.
+
+To create an ES|QL rule:
 
-.. **Aggregating query**: This is a query that uses the `STATS...BY` grouping commands. Query results cannot be matched with a particular document in Elasticsearch. For example:
+. Go to *Rules* -> *Detection rules (SIEM)* -> *Create new rule*. The *Create new rule* page displays.
+. Select **ES|QL**, then write a <<esql-rule-query-types,query>>. 
 +
-[esql]
------
+TIP: Click the help icon (image:images/esql-ref-button.png[Click the ES|QL help icon,20,20]) to open the in-product reference documentation for all ES|QL commands and functions.
++
+. Click *Continue* to <<rule-ui-basic-params, configure basic rule settings>>.
+
+[float]
+[[esql-rule-query-types]]
+==== ES|QL query types 
+
+ES|QL rule queries are loosely categorized into two types: aggregating and non-aggregating. 
+
+[float]
+[[esql-agg-query]]
+===== Aggregating query 
+
+An aggregating query uses {ref}/esql-functions-operators.html#esql-agg-functions[`STATS...BY`] functions to aggregate source event data. Generated alerts only contain the fields returned by the query.
+
+Here is an example aggregating query:
+
+[source,esql]
+----
 FROM logs*
-| STATS count = COUNT(host.name) BY host.name
-| SORT host.name
+| STATS host_count = COUNT(host.name) BY host.name
+| SORT host_count DESC
+| WHERE host_count > 20
+----
+
+- This query starts by searching logs from indices that match the pattern `logs-*`. 
+- The query then aggregates the count of events by `host.name`.
+- Next, it sorts the result by `host_count` in descending order.
+- Then, it filters events where the `host_count` field is encountered more than 20 times during the specified rule interval.
+
+NOTE: Rules that use aggregating queries might create duplicate alerts. This can happen  when events that occur in the additional look-back time are aggregated twice: once in the current rule execution and once in the a previous rule execution.
+
+[float]
+[[esql-non-agg-query]]
+===== Non-aggregating query 
+A non-aggregating query doesn't use `STATS...BY` functions and doesn't aggregate source event data. Generated alerts contain source event data and fields returned by the ES|QL query. 
+
+Here is an example of a non-aggregating query:
+[source,esql]
+-----
+FROM logs* [metadata _id, _index, _version]
```

Comment:
```suggestion
FROM logs* [metadata _id, _index, _version]
```
Should this and the previous example say `logs-*` instead of `logs*` ? The description below says `logs-*` and that's what I would expect

Reply:
This applies to all the examples below as well

Reply:
Good catch! Fixed in [cbc6444](https://github.com/elastic/security-docs/pull/4062/commits/cbc64440611a121f93d40cb3ad7d1b4f4c33dc10).